compiler/lcalc/ast.ml

```diff
@@ -134,7 +134,7 @@ let make_matchopt_with_abs_arms
   let pos = Pos.get_position @@ Bindlib.unbox arg in
   let mark : 'a -> 'a Pos.marked = Pos.mark pos in
 
-  let+ arg = arg and+ e_none = e_none and+ e_some = e_some in
```

Comment:
Yea you can't really remove these, it's a monadic bind it does not have the same syntactic sugar as regular binds.

Reply:
I just updated `ocamlformat`, we're using `0.20.1` now. On `master` when I run `make format` it doesn't change anything and the `let+ arg = arg` are conserved.